3scale/porta
# [PortaFly] Export as CSV button

**What this PR does / why we need it**:

Adds the export as CSV feature

**Which issue(s) this PR fixes** 

fixes https://issues.redhat.com/browse/THREESCALE-5324

**Verification steps** 
1. Go to Accounts -> Listing
2. Click "Export all accounts"
3. It should have downloaded an `accounts.csv` file listing the table data

**Special notes for your reviewer**:
